api/client/service/update.go

```diff
@@ -122,8 +117,8 @@ func updateService(spec *swarm.ServiceSpec, flags *pflag.FlagSet) error {
 	updateString(flagName, &spec.Name)
 	updateLabels(flags, &spec.Labels)
 	updateString("image", &cspec.Image)
-	updateSlice("command", &cspec.Command)
-	updateSlice("arg", &cspec.Command)
+	updateStringToSlice(flags, "command", &cspec.Command)
```

Comment:
So it sounds like the current behavior:

> If command is set, it ignores args entirely.

...is *not* what you have in mind.